server/terraform/modules/myhealth/main.tf

```diff
@@ -10,6 +10,19 @@ terraform {
   required_version = ">= 0.12"
 }
 
+locals {
+  # We want the storage bucket to be multi-region. Lookup the location for that
+  # using var.region prefix; based on current list at:
+  # https://cloud.google.com/storage/docs/locations
+  storage_location_prefixes = {
+    "EUROPE" = "EU",
+    "US"     = "US",
+    "ASIA"   = "ASIA"
+  }
+  storage_bucket_location = lookup(local.storage_location_prefixes,
+  upper(split("-", var.region)[0]), var.region)
+}
```

Comment:
The bucket still points to the old `var.region` location, so that needs to be updated in the terraform on line 282.

Google docs state that the location can't be changed except by destroying the bucket and creating a new bucket with the same name. There isn't a good way to do that in Terraform, so you may need to change the name in order for it to work on an existing deployment.
https://cloud.google.com/storage/docs/moving-buckets

Reply:
D'oh the merge got mangled and I didn't test after.
What happens if you move the locaiton is that Terraform will destory and re-create the bucket. GIven that this is not a common operation, the operator will first have to manually delete the contents of the bucket. Then run apply once or twice.  (Should be once but it messed up one of the dependencies when I did it, so I ran it twice and tweaked the tf file.)

Reply:
We should add a `prevent_destroy` to the storage bucket as part of this PR.

Reply:
having had to destroy/recreate the bucket in my test project a few times while testing this, i'm happy with 
force_destroy = false. if the bucket has no content, destroying it basically harmless. if it has content, force_destroy will block its destruction.